core/src/main/java/org/mskcc/cbio/portal/util/GisticReader.java

```diff
@@ -200,25 +204,26 @@ private Gistic parseLine(String line, int cancerStudyId, int chromosomeField, in
         ArrayList<CanonicalGene> genes = new ArrayList<CanonicalGene>();
         DaoGeneOptimized daoGene = DaoGeneOptimized.getInstance();
         for (String gene : _genes) {
+            
+            gene = gene.split("\\|")[0];
 
             CanonicalGene canonicalGene = daoGene.getNonAmbiguousGene(gene);
 
-            if (canonicalGene == null) {
-                canonicalGene = new CanonicalGene(gene);
-
-//                    System.out.println("gene not found, skipping: " + gene);
-//                    throw new DaoException("gene not found: " + gene);
-            }
+            if (canonicalGene != null) {
+                if (canonicalGene.isMicroRNA()) {
+                    System.err.println("ignoring miRNA: " + canonicalGene.getHugoGeneSymbolAllCaps());
```

Comment:
Please use
`ProgressMonitor.logWarning()` method. In this way all warnings are grouped together and reported at the end of the import step